### ISSUE TITLE
Mining money cannot be withdrawn

### ISSUE THREAD

**Issue by YWTURW** [RavenProject/Ravencoin#1248]
The wallet address that cannot be withdrawn for mining money is Tangem’s wallet address.

### Machine specs:
- OS:  Windows10
- CPU:  12700 F
- RAM:  D5 32GB
- Disk size: 1T
- Disk Type SDD

### Any extra information that might be useful in the debugging process.
This is normally the contents of a `debug.log` or `config.log` file. Raw text or a link to a pastebin type site are preferred.

**Comment by corviato1** [comment]
Your issue is not clear.  What specifically is the problem?

**Comment by YWTURW** [comment]
Mined coins cannot enter my cold wallet

**Comment by corviato1** [comment]
What wallet did you mine to, and what wallet are you trying to send to?

**Comment by YWTURW** [comment]
TANGEM cold wallet

**Comment by corviato1** [comment]
Is that the one you mined to, or the one you are trying to send to?

**Comment by YWTURW** [comment]
yes

**Comment by dicklarsson** [comment]
Quick note on Ravencoin's coinbase transactions: 

These are the first transactions in a block, rewarding miners with newly minted coins and transaction fees. 
However, there's a catch: these rewards can't be spent until after 100 more blocks have been mined. 
This "100-block rule" is crucial for network security, preventing potential issues like double spending

**Comment by YWTURW** [comment]
But I dug more than 500 and he hasn't put it in my wallet yet.

**Comment by dicklarsson** [comment]
Sir, are you using a mining pool, mining Ravencoin, but you cannot see the coins that you have earned in your Ravencoin address?

**Comment by YWTURW** [comment]
YES

**Comment by corviato1** [comment]
> yes

Yes to which of the two questions, and what about the unanswered question?

**Comment by YWTURW** [comment]
I used NBMiner to mine and I didn’t see any profit on Ravencoin Block Epxlorer.
https://api.ravencoin.org/address/RM5hr17144kuAYeWvvW65WywyLDczdhVsD
